test/apps/iodemo/io_demo.cc

```diff
@@ -763,6 +762,14 @@ class DemoClient : public P2pDemoCommon {
         return data_size;
     }
 
+    void close_uncompleted_servers() {
+        for (size_t i = 0; i < _active_servers.size(); ++i) {
+            if (get_num_uncompleted(i) > 0) {
+                dispatch_connection_error(_server_info[i].conn);
```

Comment:
let's have different log print in case of UCX-triggered error and IOdemo-triggered timeout